02-javascript-data-types/2-pick/index.js

```diff
@@ -5,5 +5,6 @@
  * @returns {object} - returns the new object
  */
 export const pick = (obj, ...fields) => {
-
+  const pickedEntries = Object.entries(obj).filter(([key, value]) => fields.includes(key));
+  return Object.fromEntries(pickedEntries);
```

Comment:
```suggestion
  const result = {};
  for (let field of fields) {
    if (field in obj) {
      result[field] = obj[field];
    }
  }
  return result;
```
Мы используем Object.entries(), чтобы познакомиться с entries/filter/fromEntries, или потому что решение через них лучше?
JSBench верит, что через entries медленнее: https://jsbench.me/vbkitotbly

Reply:
@konvi ответ на этот вопрос обсудим на занятии